http-endpoint-example/README.md

```diff
@@ -0,0 +1,41 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to a Amazon Web Services (AWS) Simple Notification Service (SNS) and start downloading objects.
+The running python script will accept notfications from the the SNS and download the objects according to the input paramters.
+
+
+## Prerequisites:
+
+  * API key
+  * Python 3 environment
+  * Open port
+  * A server with an Public IP or DNS address
+  * If used in production a method of verifying the messages are from AWS, this can be found [here](http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.verify.signature.html "AWS docoumentation")
+
+# Actions:
+1. ### Clone repo onto the server.
+
+2. ### Start polling the SNS by running the host.py with the following command
+````
+  python host.py <port> <api_key> <start_time> <end_time> <disagnostics)
+````
+  * The port argument is the port you want the service to run on
+  * Note that the start and end times are the forecast times (i.e. T+ times), in hours e.g. 0 to 24, up to a maximum of 144 hours (6 days).
+    * The start_time is the start of the forecast period
+    * The end_time is the end of the forecast period
+  * The disagnostics is a comma seperated list of weather you want to download
+  * The -v flag is set for verbose output.
+  * The bellow example will download all temperature and pressure objects for all time periods with verbose output and run on port 8080
+````
+  $ python host.py -v 8080 <api_key> 0 144 temperature,pressure
+    * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
+````
+
+3. ### The application will auto subscribe to the SNS
```

Comment:
This isn't true! It will automatically confirm a subscription, but the subscription has to be created manually. We need a step here that is equivalent to the [subscription step](https://github.com/MetOffice/mo-atmospheric-data-services-example-code/tree/bd1fb943aa2d72c78ef62e16921600d7db0e5998/aws-example#subscribe-the-sqs-queue-to-the-sns-topic) of the other example. In other words, now that they've got a running server they need to notify the Met Office of the URL.